server/routes/contacts/create/enter-name/createContactEnterNameSchemas.ts

```diff
@@ -3,11 +3,16 @@ import { createSchema } from '../../../../middleware/validationMiddleware'
 
 const LAST_NAME_REQUIRED_MESSAGE = "Enter the contact's last name"
 const LAST_NAME_TOO_LONG_ERROR_MSG = "Contact's last name must be 35 characters or less"
+const LAST_NAME_INVALID = "Contact's last name must not contain special characters"
 
 const FIRST_NAME_REQUIRED_MESSAGE = "Enter the contact's first name"
 const FIRST_NAME_TOO_LONG_ERROR_MSG = "Contact's first name must be 35 characters or less"
+const FIRST_NAME_INVALID = "Contact's first name must not contain special characters"
 
 const MIDDLE_NAME_TOO_LONG_ERROR_MSG = "Contact's middle name must be 35 characters or less"
+const MIDDLE_NAME_INVALID = "Contact's middle name must not contain special characters"
+
+const NAME_REGEX = /^[a-zA-Z\s,.'-]*$/
```

Comment:
Will be fine. Unless one of Frank Zappa's offspring become a contact.

Reply:
Yeah, I would prefer nothing at all TBH but did a bit of googling and the general consensus is that this accounts for most things.